### PR TITLE
Remove home from breadcrumbs

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/3572

**Current Behavior:** Home is included in the breadcrumbs.

**Desired Behavior:** Home is not included in the breadcrumbs.

This PR implements the desired behavior.

## Testing done 
Locally.

## Screenshots
image

## Acceptance criteria
- [x]"Home" is not included in breadcrumbs.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the Pre-Launch Checklist
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] A link has been provided to the VSA-QA Test Plan ticket \[use the VSA-QA Test Plan Issue Template to create the ticket within va.gov-team repo].
- [x] A link has been provided to the VSA-QA Test-Request ticket \[issue-template coming soon].
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/includes/breadcrumbs.drupal.liquid
Comment (ncksllvn) [inline src/site/includes/breadcrumbs.drupal.liquid]: Huh....Is there a story behind this? I don't see much in the issue. 

This is okay, but I'm curious on the LOE for just removing the index page from `entityUrl.breadcrumb` in Drupal, so we can just render without special cases.

Comment (kelsonic) [reply src/site/includes/breadcrumbs.drupal.liquid]: So this ticket was brought up out of a meeting with @jenniferlee-dsva and @rtwell! They will likely have more info into the reasoning why the `Home` breadcrumb should be removed! :)

Comment (kelsonic) [reply src/site/includes/breadcrumbs.drupal.liquid]: Oh! Yes, if we can do this on Drupal I would much prefer that so the FE can be dumber! 👍

Comment (kelsonic) [reply src/site/includes/breadcrumbs.drupal.liquid]: Who would we reach out to for that ask @ncksllvn?

**Comment by ncksllvn** [comment]
Not sure if it's related, but an E2E test is now failing. It might have clicked used the breadcrumbs or something. 

```
�� 526 all-claims e2e tests ��� maximal-test.json

[2019-11-21T15:44:11.080Z] 

[2019-11-21T15:44:11.080Z]     Error: failed to find element matching selector "textarea[id="root_view:worsenedFollowUp_worsenedEffects"]"

[2019-11-21T15:44:11.080Z] 

[2019-11-21T15:44:11.080Z]       93 |       await page.type(selector, `${fieldData}`);

[2019-11-21T15:44:11.080Z]       94 |       // Get the autocomplete menu out of the way

[2019-11-21T15:44:11.080Z]     > 95 |       const role = await page.$eval(selector, textbox =>

[2019-11-21T15:44:11.080Z]          |                               ^

[2019-11-21T15:44:11.080Z]       96 |         textbox.getAttribute('role'),

[2019-11-21T15:44:11.080Z]       97 |       );

[2019-11-21T15:44:11.080Z]       98 |       if (role === 'combobox') 
```

File: src/site/includes/breadcrumbs.drupal.liquid
Comment (ncksllvn) [reply src/site/includes/breadcrumbs.drupal.liquid]: I followed up in the issue to the CMS folks

**Review by ncksllvn — APPROVED** [review]
If we can't do this from Drupal, this is fine, but let's hold off on merging until we hear back.

**Review by cvalarida — APPROVED** [review]
Approving so this isn't blocked if it's required, but I agree that doing this in the CMS would be ideal. I think. :sweat_smile:

**Comment by kelsonic** [comment]
Closing this as @kevwalsh updated this on Drupal's side! 🎉

**Pull request closed without merging**
